### ISSUE TITLE
ko/docs/pytorch/0dd806f9c308d81d75b304905839ce6f59ed41fb/7a

### ISSUE THREAD

**Issue by thinker0v0** [runebookdev/runebook#885]
> <a href="#torch.cuda.amp.autocast"><code>autocast</code></a> should wrap only the forward pass(es) of your network, including the loss computation(s). Backward passes under autocast are not recommended. Backward ops run in the same type that autocast used for corresponding forward ops.

<a href="#torch.cuda.amp.autocast"> <code>autocast</code> </a> 는 손실 계산을 포함하여 네트워크의 순방향 패스 만 래핑해야합니다. 자동 캐스트에서 역전파(backpropagation)는 권장되지 않습니다. 역전파 프로세스는 해당 순방향 작업에 사용 된 자동 캐스팅과 동일한 유형으로 실행됩니다.
